tests/browser/index.js

```diff
@@ -1,23 +0,0 @@
-require('./spec/XAxis.spec');
-require('./spec/YAxis.spec');
-require('./spec/YAxisLabels.spec');
-require('./spec/XAxisLabels.spec');
-require('./spec/XAxisTitle.spec');
-require('./spec/YAxisTitle.spec');
-const enzyme = require('enzyme');
-const adapter = require('enzyme-adapter-react-16');
-
-// some tests must be run in a browser environment
-// also it can be easier to debug tests in browser thanks to chrome debugger
-// place browser tests in tests/browser/spec and run `npm run test-browser`
```

Comment:
I'm a fan of having some browser-based tests for these reasons. It's easy to debug in the Chrome debugger. Also since this is a dataviz library that interacts with canvas a lot it's better not to have used the mocked and inaccurate version of measureText. What do you think about keeping the browser based tests using mocha for now and reducing this PR to only migrate the JSDOM tests to jest for now. We can revisit the browser based tests later on. @PRTTMPRPHT and @ekh64

Reply:
Yeah, I can understand that! Then I'll revert the changes for the concerned tests and add back the dependencies. I'm currently on vacation, so I will do that once I'm back on July 1st.